05-promoting_diversity.Rmd

```diff
@@ -148,30 +149,33 @@ Recent projects in which participants are recruited through community engagement
 1) Provide adequate financial, logistical, and time support
 
 Researchers should consider how participants can cope with healthcare needs following participation in a study.
-Investigators should consider how to support caregivers who have important obligations to tend to their family members or others, for example when possible, perhaps home visits would be feasible. Other strategies include providing flexible hours for trial visits and support for transportation, such as payment for a rideshare service or taxi.[@ACRP_representation_2020] Researchers should also consider providing patients with a cell phone to improve communication.[@clark_increasing_2019]
+Investigators should consider how to support caregivers who have important obligations to tend to their family members or others, for example when possible, perhaps home visits would be feasible.
+Other strategies include providing flexible hours for trial visits and support for transportation, such as payment for a rideshare service or taxi [@ACRP_representation_2020]. Researchers should also consider providing patients with a cell phone to improve communication [@clark_increasing_2019].
 
-2) More promotion of health care access in general 
+2) More promotion of health care access in general
 
-Individuals have been shown to be more likely to participate when advised by a health care provider.[@clark_increasing_2019] Increasing health care system access in general could therefore improve clinical trial participation.
+Individuals are more likely to participate when advised by a health care provider [@clark_increasing_2019].
+Increasing health care access in general could therefore improve clinical trial participation.
 
 3) More promotion of community engagement
 
-Community stakeholders can help to assess if recruitment information is culturally appropriate and can help with creating more equitable recruitment strategies. 
+Community stakeholders can help assess if recruitment information is culturally appropriate and can help create more equitable recruitment strategies.
 
 4) More inclusive research teams
 
-Individuals have been shown to be more trusting when research teams include members that are similar to themselves. [@griffith_determinants_2020]
+Individuals have been shown to be more trusting when research teams include members that are similar to themselves [@griffith_determinants_2020].
 
 5) More inclusive practices
 
-When obtaining information about gender, researchers should collect more informative information such as the model proposed in @chen_approach_2019. Similarly, more detailed information about ancestry could also lead to more informative findings about specific populations. 
+When obtaining information about gender, researchers should collect more informative information such as the model proposed in @chen_approach_2019.
+Similarly, more detailed information about ancestry could also lead to more informative findings about specific populations.
```

Comment:
This feels like a separate point from the one before it. Can more info be added about both of these points?